src/Features/Core/Portable/ExtractMethod/MethodExtractor.CodeGenerator.cs

```diff
@@ -165,10 +166,10 @@ SyntaxNode InsertLocalFunction()
 
                 SyntaxNode InsertNormalMethod()
                 {
-                    var syntaxKinds = document.GetLanguageService<ISyntaxKindsService>();
+                    var syntaxKinds = document.GetRequiredLanguageService<ISyntaxKindsService>();
```

Comment:
>GetRequiredLanguageService [](http://example.com/codeflow?start=47&length=26)

I would have expected this to have been done in response to removing the nullability pragma at the beginning of the file.

Reply:
yeah, i think i started removing nullability, then added back as it was too large.